documentation/modules/exploit/windows/http/diskboss_get_bof.md

```diff
@@ -0,0 +1,96 @@
+## Vulnerable Application
+
+DiskBoss Enterprise versions up to v7.5.12 are affected by a stack-based buffer overflow vulnerability which can be leveraged by an attacker to execute arbitrary code in the context of NT AUTHORITY\SYSTEM on the target. The vulnerability is caused by improper bounds checking of the request path in HTTP GET requests sent to the built-in web server. This module has been tested successfully on Windows XP SP3 and Windows 7 SP1.
```

Comment:
I would link "DiskBoss Enterprise" to the company, and at the end, provide a link to the EDB for the vuln software.